### ISSUE TITLE
Outlook desktop client: Add in instances freeze after add in has been pinned.

### ISSUE THREAD

**Issue by jonmoshier** [OfficeDev/script-lab#821]
### Bug Report

- **Host**: Outlook
- **OS**: Windows 10
- **Desktop Outlook Client**: Microsoft Outlook for Microsoft 365 MSO (16.0.12730.20252) 64 Bit

#### Expected behavior:

Add in should be responsive to user input.
 
#### Actual behavior:

Add in is not responsive to user input.

#### Steps to Reproduce:

1. Open outlook desktop client.
2. Go to inbox and select an email.
3. Open the Script Lab add in.

Addin State:  Loads and is responsive to user interaction.

4. Pin add in.
5. Click Reply on email.

Addin State: Add in closed

6. Reopen Script Lab Run Add-in

Add in state: UI is not responsive to user input.

Video: [Inbox - jonmoshier_filevine@outlook.com - Outlook (Logging Enabled) 2020-06-24 17-46-23.zip](https://github.com/OfficeDev/script-lab/files/4828377/Inbox.-.jonmoshier_filevine%40outlook.com.-.Outlook.Logging.Enabled.2020-06-24.17-46-23.zip)

#### Context

Our team is working on our own add in and is seeing this same issue with the Outlook Desktop Client and add in's freezing. I added Script Lab to test this scenario and see that this is an issue for your app as well.

If there's a better place to post this bug (outlook desktop, outlook.js), please let me know.

**Comment by TCourtneyOwen** [comment]
Thanks for reporting this issue. I will see if I can repro the behavior and get back to you

-Courtney

**Comment by TCourtneyOwen** [comment]
@jonmoshier I am unable to repro this behavior and wanted to gather some additional information.  When you run step #6 are you still in Outlook compose mode in the mail you are replying on? Or do you return back to Read mode and then try to run step #6. 

@JuaneloJuanelo @nokafor can you see if you repro this behavior?

Thanks,

Courtney

**Comment by jonmoshier** [comment]
@TCourtneyOwen I've attached a zip file that contains a screen recording of the issue.

**Comment by jonmoshier** [comment]
I have an issue at https://github.com/OfficeDev/office-js/issues/1239 as well and they added this link -> https://developer.microsoft.com/en-us/office/blogs/microsoft-edge-webview-for-office-add-ins/

I think my version of outlook is using ie 11 and yours is using edge, perhaps that's the difference.

I'm on Windows 10 Pro 1904 but from the article; Edge isn't being used by non-subscribed users of outlook and I am testing with a non-subscribed user.

I'll try and test with a subscribed user tomorrow.

**Comment by jonmoshier** [comment]
I tried with a subscribed user and this issue still persists. Perhaps there are some other things I can check?

**Comment by TCourtneyOwen** [comment]
I just switched my webview to use IE and I still can't repro this. I've tried IE, Edge and Edge Chromium now and still can't repro this. I am not questioning that you are indeed hitting a real issue, I just haven't been able to repro it yet.  

Can you please try the following:  delete the contents of  %:\Users\<user-name>\AppData\Local\Microsoft\Office\16.0\Wef on your local machine and try opening and closing Outlook and running your scenario again

**Comment by jonmoshier** [comment]
Yes, just completed that and I'm still hitting the issue. We have multiple people on our team that are able to reproduce this issue with different add in's, not just script lab, and we're concerned about how its going to impact our own add in user experience.

Any chance we could schedule a call and demonstrate this for you, maybe we'd be able to offer some more context. And even if this is an error on our part, maybe we could help mitigate this issue for our own add in users.

Thanks for your help!

**Comment by TCourtneyOwen** [comment]
Just to clarify, you are hitting this with other add-ins from the store other than Script Lab? If so, can you please let me know what those add-ins are so I can try them as well?  Is this specific to Outlook, or are you seeing this behavior in other hosts such as Excel and Word?

**Comment by jonmoshier** [comment]
Yes, we're seeing this behavior with the Sales Force add in as well. We stumbled onto this issue when testing our add in in development and wanted to see if other add ins had the same issue (they do).

This is on a teammates PC, he's using Microsoft Outlook for Microsoft 365 MSO (16.0.12827.20328) 64-bit

[Salesforce-add-in-pin-bug-outlook-client.zip](https://github.com/OfficeDev/script-lab/files/4833579/Salesforce-add-in-pin-bug-outlook-client.zip)

**Comment by TCourtneyOwen** [comment]
@jonmoshier I have looped in some Outlook Engineers on this issue as well. I suspect they will be reaching out soon.

-Courtney

**Comment by jonmoshier** [comment]
Thank you Courtney, I realize this issue left the scope of Script Lab and I appreciate your help along the way.

Script Lab is awesome by the way, we've started using it as a way to explore the office context, like a built in REPL and it's really a great tool and open source project. 

Thanks for all of your help!

**Comment by TCourtneyOwen** [comment]
Sure thing, I hope we can find the cause and a solution for this issue.

-Courtney

**Comment by ElizabethSamuel-MSFT** [comment]
Closing all issues since this repo is being archived and no longer maintained.
